Solutions/Cyware/Hunting queries/MatchCywareIntelWatchlistItemsWithCommonLogs.yaml

```diff
@@ -0,0 +1,26 @@
+id: 61c99147-b749-4164-80b1-c4bfa4efa704
+name: Match Cyware Intel Watchlist Items With Common Logs
+description: |
+  'Query to match common security log identifiers with IOCs held by the Cyware Intel watchlist that is created automatically by Cyware'
+requiredDataConnectors: [
+'DeviceProcessEvents'
+]
+tactics:
+  - Command And Control
```

Comment:
Hi @AashiqRamachandran, tactics names MUST NOT have any spaces. It should be `CommandAndControl`. Also relevantTechniques missing for this tactic.